Buildings/Controls/OBC/Utilities/BaseClasses/Validation/NormalizedTimeDelay.mo

```diff
@@ -0,0 +1,38 @@
+within Buildings.Controls.OBC.Utilities.BaseClasses.Validation;
```

Comment:
The folder `Buildings/Controls/OBC/Utilities/BaseClasses/Validation` should be removed if it is not needed.

Reply:
This folder is removed.